webmention.py

```diff
@@ -67,6 +68,14 @@ def discover(url, **requests_kwargs):
       logger.debug(f'Webmention discovery: got endpoint in tag: {endpoint}')
       return Endpoint(endpoint, resp)
 
+  http_equiv = util.fetch_http_equiv(soup)
+  if http_equiv: # else implicit break out and continue like normal
+    endpoint = util.fragmentless(urljoin(url, http_equiv))
+    logger.debug(f'Webmention discovery: got http_equiv in tag: {endpoint}')
```

Comment:
Minor, consider removing this? It's a bit misleading when `follow_meta_refresh` is `False`.

Reply:
removed the debug log outside of the follow_meta_refresh check